site/docs/reference/Connectors/capture-connectors/google-firestore.md

```diff
@@ -59,15 +89,43 @@ captures:
             "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/service-account-email"
     bindings:
       - resource:
+          #The below `path` will capture all Firestore documents that match the pattern
+          #`orgs/<orgID>/runs/<runID>/runResults/<runResultID>/queryResults`.
+          #See the Data Model section above for details.
           path: orgs/*/runs/*/runResults/*/queryResults
+          backfillMode: async
         target: ${PREFIX}/orgs_runs_runResults_queryResults
       - resource:
           path: orgs/*/runs/*/runResults
+          backfillMode: async
         target: ${PREFIX}/orgs_runs_runResults
       - resource:
           path: orgs/*/runs
+          backfillMode: async
         target: ${PREFIX}/orgs_runs
       - resource:
           path: orgs
+          backfillMode: async
         target: ${PREFIX}/orgs
-```
\ No newline at end of file
+```
+
+## Backfill mode
+
+In each captured collection's [binding configuration](#bindings), you can choose whether and how to backfill historical data.
+There are three options:
+
+* `none`: Skip preexisting data in the Firestore collection. Capture only new documents and changes to existing documents that occur after the capture is published.
+
+* `sync`: Request that Firestore stream all changes to the collection since its creation, in order.
+
+   This mode provides the strongest guarantee against duplicated data, but can cause errors for large datasets as Firestore may terminate the process after about ten minutes.
+
+* `async`: Use two threads to capture data. The first captures new documents, as with `none`.
+The second progressively ingests historical data in chunks.
+
+   This mode is most reliable for Firestore collections of all sizes but provides slightly weaker guarantees against data duplication.
```

Comment:
Maybe a dumb suggestion, but merging this sentence with the previous line would make the "async" section only two paragraphs, which might (correctly) make it feel a bit simpler than the "sync" option if we add some additional "here's exactly how that can go wrong" details there.